xbmc/guilib/GUIEditControl.cpp

```diff
@@ -538,7 +537,7 @@ void CGUIEditControl::ProcessText(unsigned int currentTime)
     if (HasFocus() || leftTextWidth == 0)
       changed |= m_label2.SetOverflow(CGUILabel::OVER_FLOW_CLIP);
     else
-      changed |= m_label2.SetOverflow(CGUILabel::OVER_FLOW_TRUNCATE);
+      changed |= m_label2.SetOverflow(CGUILabel::OVER_FLOW_TRUNCATE_LEFT);
```

Comment:
Doesn't left or right truncation depend on the offset of the first character displayed?
Worst case there could be both left and right truncation. probably best left for another PR though, this one does not mean to change the behavior.

Reply:
no not good, better as is otherwise break the overflow behaviour